src/test/java/api/loans/scenarios/CheckInDeclaredLostItemTest.java

```diff
@@ -0,0 +1,477 @@
+
+    verifyLostItemFeeAccount(isAccountClosed(itemFee));
+    verifyLostItemFeeAccountAction(isCloseActionCreated(itemFee));
+
+    verifyLostItemProcessingFeeAccount(isAccountClosed(processingFee));
+    verifyLostItemProcessingFeeAccountAction(isCloseActionCreated(processingFee));
+  }
+
+  @Test
+  public void shouldRefundTransferredFee() {
+    final double setCostFee = 10.89;
+    final double processingFee = 5.00;
+
+    useChargeableRefundableLostItemFee(setCostFee, processingFee);
+
+    declareItemLost();
+
+    final JsonObject lostItemFeeAccount = feeFineAccountFixture
+      .getLostItemFeeAccount(loan.getId());
+    feeFineAccountFixture.transfer(lostItemFeeAccount.getString("id"), setCostFee);
+
+    checkInFixture.checkInByBarcode(item);
+
+    verifyLostItemFeeAccount(isAccountRefundedFully(setCostFee));
+    verifyLostItemFeeAccountAction(areTransferRefundActionsCreated(setCostFee));
+
+    verifyLostItemProcessingFeeAccount(isAccountClosed(processingFee));
+    verifyLostItemProcessingFeeAccountAction(isCloseActionCreated(processingFee));
+  }
+
+  @Test
+  public void shouldRefundPaidFee() {
+    final double setCostFee = 9.99;
+    final double processingFee = 5.00;
+
+    useChargeableRefundableLostItemFee(setCostFee, processingFee);
+
+    declareItemLost();
+
+    final JsonObject lostItemFeeAccount = feeFineAccountFixture
+      .getLostItemFeeAccount(loan.getId());
+    feeFineAccountFixture.pay(lostItemFeeAccount.getString("id"), setCostFee);
+
+    checkInFixture.checkInByBarcode(item);
+
+    verifyLostItemFeeAccount(isAccountRefundedFully(setCostFee));
+    verifyLostItemFeeAccountAction(arePaymentRefundActionsCreated(setCostFee));
+
+    verifyLostItemProcessingFeeAccount(isAccountClosed(processingFee));
+    verifyLostItemProcessingFeeAccountAction(isCloseActionCreated(processingFee));
+  }
+
+  @Test
+  public void shouldRefundPaidAndTransferredFee() {
+    final double transferAmount = 6.0;
+    final double paymentAmount = 4.0;
+    final double setCostFee = transferAmount + paymentAmount;
+    final double processingFee = 5.00;
+
+    useChargeableRefundableLostItemFee(setCostFee, processingFee);
+
+    declareItemLost();
+
+    final JsonObject lostItemFeeAccount = feeFineAccountFixture
+      .getLostItemFeeAccount(loan.getId());
+    feeFineAccountFixture.transfer(lostItemFeeAccount, transferAmount);
+    feeFineAccountFixture.pay(lostItemFeeAccount, paymentAmount);
+
+    checkInFixture.checkInByBarcode(item);
+
+    verifyLostItemFeeAccount(isAccountRefundedFully(setCostFee));
+    verifyLostItemFeeAccountAction(areTransferRefundActionsCreated(transferAmount));
+    verifyLostItemFeeAccountAction(arePaymentRefundActionsCreated(paymentAmount));
+
+    verifyLostItemProcessingFeeAccount(isAccountClosed(processingFee));
+    verifyLostItemProcessingFeeAccountAction(isCloseActionCreated(processingFee));
+  }
+
+  @Test
+  public void shouldCloseAndRefundPaidAndTransferredFee() {
```

Comment:
What is the difference between this and `shouldRefundPaidAndTransferredFee`?

Reply:
This test about partial transfer and payment, but `shouldRefundPaidAndTransferredFee` is about full payment/transfer.

Reply:
Ok, thanks. 

After a quick scan of the story, I couldn't find a definition for what closing a partially paid was called. What is the domain term for that, is it cancel?

(I may have asked the same question in another comment, feel free to only reply to one)

Reply:
Based upon the answer to another question, which referred to the remaining amount being cancelled, I suggest renaming this. What do you think?

```suggestion
  public void shouldRefundPartiallyPaidAndTransferredFeeAndCancelRemainingAmount() {
```

Reply:
I haven't seen any special term for cancellation when a fee is partially paid. I use `cancellation` for all the cases.

Reply:
> I haven't seen any special term for cancellation when a fee is partially paid. I use `cancellation` for all the cases.

Then I think this test name should also use cancellation as well.